src/helpers.ts

```diff
@@ -79,7 +79,8 @@ export function pathToJest({ pathToJest, rootPath }: IPluginResourceSettings) {
     return 'npm test --'
   }
 
-  return getLocalPathForExecutable(rootPath, 'jest') || 'jest' + nodeBinExtension
+  const p = getLocalPathForExecutable(rootPath, 'jest') || 'jest' + nodeBinExtension
```

Comment:
I don't have it in mind, if we always use the shell, do we need `nodeBinExtension` on Windows?

Reply:
I don't have a windows machine to verify, if it worked without, I can remove it... does it matter which shell is used? I could imagine users might want to pass their favor shell in the future...